tests/unit_tests.py

```diff
@@ -304,13 +304,15 @@ def test_kernels_g_pull(self):
 
   def test_competition_a_list(self):
     try:
-      competitions = api.competitions_list()
-      self.assertGreater(len(competitions),
-                         0)  # Assuming there should be some competitions
+      competitions = api.competitions_list(group='general')
+      self.assertGreater(len(competitions), 0)
+      self.assertLess(len(competitions), 20)
```

Comment:
nit - I think this should be `assertLessEqual`?  Assuming the page size is 20, it should be possible in the future to get 20, depending on how many are active.